ocs_ci/utility/utils.py

```diff
@@ -2536,6 +2540,15 @@ def ceph_health_check_base(namespace=None):
     namespace = namespace or config.ENV_DATA["cluster_namespace"]
     health = run_ceph_health_cmd(namespace)
 
+    if "daemons have recently crashed" in health and resolve_daemon_crash:
```

Comment:
I would move this block under to separate function like ceph_health_resolve_daemon_crash. Than have function called `fix_ceph_health(health_status)` which will take the health as param and then it will try to fix that, so we can be adding multiple if and elif blocks under it. Which will decide which function to call to fix our issue.